### ISSUE TITLE
Postgres migrations are inconsistent on alternate schema

### ISSUE THREAD

**Issue by Tomsons** [n8n-io/n8n#3325]
**Describe the bug**
The current migrations are inconsistent, which means that there is no way to start a new n8n instance on a clean Postgres DB with an alternate schema. Following your docker-compose example produces the following output:

```
Initializing n8n process
UserSettings were generated and saved to: /root/.n8n/config
query is slow: SELECT * FROM "n8n"."migrations" "migrations"  ORDER BY "id" DESC
execution time: 3234
There was an error initializing DB: "relation "credentials_entity" does not exist"

Stopping n8n...
```

**To Reproduce**
1. Follow the docker-compose example for Postgres by using a schema other than `public`

**Expected behavior**
All migrations should run as expected and the application should startup

**Environment (please complete the following information):**
 - OS: [e.g. Ubuntu Linux 18.04]
 - n8n Version: docker digest 403a10ae480e18502eb80ca6490a5ba54555251911ff91c1766dd6c30987fdaa
 - Node.js Version: the container's
 - Database system: Postgres
 - Operation mode: own

**Recommendations**
- [ ] Provide an extra cli command to run only the migrations 
- [ ] Introduce an extra CI check to ensure the all migrations for all db engines are working as expected

**Comment by ivov** [comment]
Thank you for your report.

Busy with other tasks at the moment, but from a quick look it seems to be the whitespace before the tablename:

https://github.com/n8n-io/n8n/blob/29ddac30d33caff1cf8a061d619742fdb3402d49/packages/cli/src/databases/postgresdb/migrations/1646992772331-CreateUserManagement.ts#L134

**Comment by janober** [comment]
Got fixed with #3356 and will be released with the next version.

**Comment by janober** [comment]
Fix got released with `n8n@0.179.0`
